### ISSUE TITLE
Grant access to reports for recipients which account have been disabled

### ISSUE THREAD

**Issue by evariitta** [globaleaks/GlobaLeaks#3219]
With the latest updates, a recipient can grant access to a report for a recipient which account has been disabled.

I think this is quite confusing, for it leads to understand that these disabled users would have ongoing access to the channel.   

**Describe the solution you'd like** 

I would like that disabled recipients wouldn't be shown in the list of recipients when granting user access to reports.

**Comment by evilaliv3** [comment]
Thank you for your feedback @evariitta 

The "disabled" state of an account does currently disable only the possibility of login of a recipient and is to be used for security reasons when the recipient is on temporarily leave to limit access by malicious users during this period.
Is for this reason that disabled recipients does currently receive reports and can be assigned to reports during their leave so that at their return they could check on them.

Based on this considerations do you still consider that a disabled user should not be listed during their temporarely leave or would you find it useful to leave it visible so that their colleagues can prepare assignment to be ready at their return?

I include here some of the other users that could be interested in the discussion @elbill @maxmois @giorgiofraschini @larrykind

**Comment by evariitta** [comment]
I see this point. We have disabled also users that has left the workplace. This leaves us a record on people who have ever had access to the reports.
We can of course change our way of work, if it is meaning to use this only for temporary leaves and you consider it to be useful to have them on the list all the way.

**Comment by larrykind** [comment]
Hello everyone, 
thank you for mention me. Agree with @evilaliv3, so in this case maybe a simple label changing should solve. In this case "user disabled" could be replaced by "login disabled". 
Anyway I could suggest a feature to completely "deactivate" a receiver account, making the user not selectable for report sharing: that deactivated user could not login, it will never receive the reports nor emails from that moment on, until it will be reactivated from an admin. In fact actually even if you

**Comment by elbill** [comment]
Does a disabled user receive notifications if they are included in a context?
I would agree with @larrykind that there should be some indication when sharing a report and also in the report itself that a user is disabled. A complete deactivation would be also useful.

**Comment by evilaliv3** [comment]
Thank you everyone for your feedback.

Yes currently disabled recipients receive notifications about the arrival or update of a report; this is intended so that they can eventually be aware of what is happening on the platform even if they are temporarely out of service.

**Comment by giorgiofraschini** [comment]
I agree with this. A disabled user should be kept updated for when he will be enabled again. If it is not going to be enabled again, it should be deactivated.

**Comment by evariitta** [comment]
So could there be an option to deactivate a user? 
Or should we, for now, delete the users who aren't intented to be enabled/activated again?

**Comment by evilaliv3** [comment]
> So could there be an option to deactivate a user?
yes definitely, think there is consensus of everybody on this. 

What we shall identify is clearly how to call the features to clearly differentiate them and decide well what do.
As Disable and Deatctivate are synonims probably we should clearly name one of these two feature with an explicit text like "Disable the user login". In this way we will ass just one new text to the set of texts to be translated/internationalized keeping low the volunteer need for translations.

>Or should we, for now, delete the users who aren't intented to be enabled/activated again?
Yes for the moment you should do this.
I'm sorry there is not a fast alternative. As the software is opensource and community driven, for this as for any suggestion you may propose, please feel free to ask your developers team if they could support the development of the feature.

**Comment by tiina-s** [comment]
Hi @evilaliv3 ! Sorry to make a comment for so old post, but I was just thinking about this matter and found these old comments. I was just wondering, if the situation is still like this, that if user is disabled by us, will they still receive notifications from the channel?

**Comment by evilaliv3** [comment]
Hey @tiina-s , thank you for your question that is actually very pertinent and reminds me that i wanted to check in also on the initial topic opened by Eva.

Regarding your question, users that are disabled by you, unless a bug exist, should not receive any notification.
